book/mvc.md

```diff
@@ -1,31 +1,30 @@
 
-The problem with MVC is that, despite its simplicity, it's not so easy to tell what "Model", "View" and "Controller" really mean. Let's find out starting with the simplest one.
+A key issue for beginners with MVC is that despite its simplicity, it's not always easy to identify what a "Model", "View", and "Controller" really mean. Let's find out by starting with the simplest one.
 
 
 ## Controller
 
-The role of the controller is to accept input and convert it to commands for the model or view. So basically controller works with external data and environment such as:
+The role of the controller is to accept input and convert it to commands for the model or view. Essentially the controller works with external data and environment such as:
 
 - GET, POST and other requests in web
 - user input in console
 
 ## View
 
-A view layer is getting prepared data from controller and formats it before sending it to the user. It can create HTML or JSON or whatever format is needed.
+The view layer processes and formats data from the controller before sending it to the user. It generates HTML, JSON, or whatever format is needed.
 
 > Note: It is strictly forbidden to work with any environment, database or user input directly in the view. It should be in controller.
 
 ## Model
 
-A model is the most interesting part of the MVC and the most misunderstood one. MVC model is often confused with Yii Model class and even Active Record. These are not the same thing. MVC model is not a single class, it's the whole domain layer (also called problem layer or application logic). Given data from controller it's doing actual job of the application and passing results back to controller.
+The model is the most interesting part of the MVC pattern and the most misunderstood one. A MVC model is often confused with the Yii Model class and even Active Records. These are not the same thing. The MVC model is not a single class, it's the whole domain layer (also called the problem layer or application logic). Given data from the controller it's doing actual work of the application and passing results back to controller.
 
-> Note: Model in MVC is not Yii Model class and is not Active Record. It can be totally disconnected from the database structure.
+> Note: Its possible the model can be totally disconnected from the database structure.
 
-ActiveRecord classes should not contain any significant business logic. It deserves to be in separate classes which are built according to [SOLID](solid.md) and [Dependency Inversion](dependencies.md). Don't be afraid to create your own classes which are not inherited from anything from the framework.
+Yii ActiveRecord classes should not contain any significant business logic. It deserves to be in separate classes which are built according to [SOLID](solid.md) and [Dependency Inversion](dependencies.md). Don't be afraid to create your own classes which are not inherited from anything from the framework.
```

Comment:
I'd leave it as is. Adding `Yii` makes it sound like "you may extend from Yii's AR class and put your business logic there" while it's not a good thing to do.